.github/workflows/publish.yaml

```diff
@@ -68,6 +68,8 @@ jobs:
         run: eas build --non-interactive --platform android --profile preview
 
   build-ios:
+    if: ${{ false }} # disable iOS builds until we migrate to a different Expo tier
+
```

Comment:
iOS builds are hanging and reducing our ability to run concurrent builds

Reply:
@floatplane let's make sure this gets surfaced this week in the mtg